sdk/formrecognizer/ai-form-recognizer/test/utils/recordedClients.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Does `utils` need to be moved inside `public` so min/max testing can work?

Reply:
I don't believe so.

Reply:
Let's make sure to run the min/max pipeline to make sure everything is in order before merging.

Reply:
minmax passed on the current commit